cc_core.c

```diff
@@ -2710,6 +2710,7 @@ void statement(void)
 		global_token = global_token->next;
 	}
 	else if((NULL != lookup_type(global_token->s, prim_types)) ||
+			match("enum", global_token->s) ||
```

Comment:
please fix whitespace (as you have already mentioned earlier) to be consistent with what is already in the file.

Reply:
Fixed.